service/backend/pchain/account.go

```diff
@@ -428,29 +446,3 @@ func (b *Backend) getAccountUTXOs(ctx context.Context, addr ids.ShortID, sourceC
 
 	return utxos, nil
 }
-
-func (b *Backend) processUtxos(currencyAssetIDs map[ids.ID]struct{}, utxos []avax.UTXO) ([]*types.Coin, error) {
```

Comment:
I know there is a specular methods in cchainatomic backend, but I feel this method is not really doing that much and it does not really need to skip currencies since they are already filtered out. Absorbed this method into caller